source/extensions/filters/common/ext_authz/ext_authz.h

```diff
@@ -53,6 +53,8 @@ struct Response {
   Http::HeaderVector headers_to_append;
   // Optional http headers used on either denied or ok responses.
   Http::HeaderVector headers_to_add;
+  // Optional http headers used on either denied or ok responses.
+  Http::HeaderVector headers_to_add_and_append;
```

Comment:
Nit. `headers_to_append`, and please update the comment.

Reply:
it seems there is a variable named `headers_to_append` above